StarterKit/Session2/agenda.md

```diff
@@ -61,18 +61,21 @@ Reflect also on what kind of knowledge you used in this exercise:
 
 The goal of this exercise is to be a concrete thing to *do* when looking at new code for the first time. New code can be scary, doing something will help! Digital annotation can take a bit longer than on paper. Leave time for your group members to settle on a technique that works for them.
 
-### (10 mins) Independently - Examine structure
+### (12 mins) Independently - Examine structure
```

Comment:
should this be 10 or 12?

Reply:
I can't decide. Definitely 12 for the 1st session. After that 10 might be ok. It's also down to the length of the code. In some ways it's a shame ot waste more than 10 mins.